mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -166,6 +169,38 @@ def __init__(self, name=None, bus=None, use_settings=True):
         # Skill Public API
         self.public_api = {}
 
+        if bus:
+            self._startup(bus)
```

Comment:
no, this is always called by the skill loader, to be called here it needs adoption from mycroft-core to tell future skills to expose the bus in args, since the original PR is stale i do not expect that to happen

this is here because it replaces the previous call to self.bind(bus) which also wasnt called 99% of the times